src/components/VDate/datepicker.js

```diff
@@ -183,16 +183,22 @@ DatePicker.prototype.setFocusDay = function(flag) {
 
   var fd = this.focusDay;
 
-  function checkDay(d) {
-    d.domNode.setAttribute('tabindex', '-1');
+  /**
+   * [checkDay description]
+   *
+   * @param  {object}    day [description]
+   * @return {undefined}     Mutates day object
+   */
+  function checkDay(day) {
+    day.domNode.setAttribute('tabindex', '-1');
     if (
-      d.day.getDate() == fd.getDate() &&
-      d.day.getMonth() == fd.getMonth() &&
-      d.day.getFullYear() == fd.getFullYear()
+      day.day.getDate() == fd.getDate() &&
```

Comment:
Renamed single character variable because it took me longer than it should have to understand what it represented.

Reply:
Yeah, no problem. I followed someone else's example on that part, but dont have any reason to keep it 1 char. Clarity is better.